themes/bootstrap3/css/vendor/cookieconsent.css

```diff
@@ -0,0 +1 @@
transform:translateX(100%)}#cc-main .pm--bar.pm--wide{max-width:35em}#cc-main .pm--bar.pm--wide .pm__body,#cc-main .pm--bar.pm--wide .pm__footer,#cc-main .pm--bar.pm--wide .pm__header{padding:1em 1.65em}#cc-main .pm--bar.pm--wide .pm__btn-group:nth-child(2),#cc-main .pm--bar.pm--wide .pm__btn:nth-child(2){margin:0 0 0 6px}#cc-main .pm--bar.pm--wide.pm--flip .pm__btn-group,#cc-main .pm--bar.pm--wide.pm--flip .pm__footer{flex-direction:row-reverse}#cc-main .pm--bar.pm--wide.pm--flip .pm__btn-group:nth-child(2),#cc-main .pm--bar.pm--wide.pm--flip .pm__btn:nth-child(2){margin:0 6px 0 0}#cc-main .pm-wrapper:before{background:rgba(0,0,0,.541);background:var(--cc-overlay-bg);bottom:0;content:"";left:0;opacity:0;position:fixed;right:0;top:0;visibility:hidden;z-index:1}#cc-main.c--anim .pm,#cc-main.c--anim .pm-wrapper:before{transition:opacity .3s ease,visibility .5s ease,-webkit-transform .5s cubic-bezier(.34,.98,.44,1);transition:opacity .3s ease,visibility .5s ease,transform .5s cubic-bezier(.34,.98,.44,1);transition:opacity .3s ease,visibility .5s ease,transform .5s cubic-bezier(.34,.98,.44,1),-webkit-transform .5s cubic-bezier(.34,.98,.44,1)}.show--preferences #cc-main.c--anim .pm{opacity:1;visibility:visible!important}.show--preferences #cc-main.c--anim .pm--box{-webkit-transform:translateY(-50%);transform:translateY(-50%)}.show--preferences #cc-main.c--anim .pm--bar{-webkit-transform:translateX(0);transform:translateX(0)}.show--preferences #cc-main.c--anim .pm-wrapper:before{opacity:1;visibility:visible}@media screen and (max-width:640px){#cc-main .pm{border-radius:0;bottom:0;height:auto;left:0;max-height:100%;max-width:none;right:0;top:0;-webkit-transform:translateY(30px) scale(.98);transform:translateY(30px) scale(.98)}#cc-main .pm__body,#cc-main .pm__footer,#cc-main .pm__header{padding:1em 1.15em!important}#cc-main .pm__badge{display:none}#cc-main .pm__section-table,#cc-main .pm__table-body,#cc-main .pm__table-td,#cc-main .pm__table-th,#cc-main .pm__table-tr{display:block}#cc-main .pm__table-head{display:none}#cc-main .pm__table-tr{border-top:1px solid #f0f4f7;border-top:1px solid var(--cc-separator-border-color)}#cc-main .pm__table-td{display:flex;justify-content:space-between}#cc-main .pm__table-td:before{color:#2c2f31;color:var(--cc-primary-color);content:attr(data-column);flex:1;font-weight:600;min-width:100px;overflow:hidden;padding-right:2em;text-overflow:ellipsis}#cc-main .pm__table-td>div{flex:3}#cc-main .pm__btn-group,#cc-main .pm__footer{flex-direction:column!important}#cc-main .pm__btn-group{display:flex!important}#cc-main .pm__btn-group:nth-child(2),#cc-main .pm__btn:nth-child(2){margin:6px 0 0!important}#cc-main .pm--flip .pm__btn-group,#cc-main .pm--flip .pm__footer{flex-direction:column-reverse!important}#cc-main .pm--flip .pm__btn-group:nth-child(2),#cc-main .pm--flip .pm__btn:nth-child(2){margin:0 0 6px!important}.show--preferences #cc-main.c--anim .pm{-webkit-transform:translateY(0)!important;transform:translateY(0)!important}}
```

Comment:
Should this file be copied in by Phing as well?

Reply:
No, it's covered by the new less files that give more control over the colors and make it blend with VuFind better.

Reply:
I adopted the less files into the core so they're included with `grunt less`. I wouldn't consider them third-party anymore since they're pretty VuFind-specific.